project_event/models/project_project.py

```diff
@@ -30,6 +30,30 @@ class Project(models.Model):
     notes = fields.Text(
         string='Notes',
     )
+    state = fields.Selection([
```

Comment:
@eilst :     _description = __doc__  can be included in the class header

Reply:
Done